config/dsConfig.js

```diff
@@ -1,6 +1,3 @@
 module.exports = {
   baseURL: process.env.DS_API_URL,
```

Comment:
does removing the `process.env.DS_API_TOKEN` break anything in the app? Were we not relying on this being defined and exported out of the config here?